### DISCUSSION TITLE
Make sure to download the checkpoint and place ckpt-0

### DISCUSSION THREAD

**Discussion by kmcowan** [xai-org/grok-1#151] [Q&A] [answered]
I don't see ckpt-0 in the repo or on huggingface.com.  Where do I download it?

**Comment by elysiumplain** [comment, marked as answer]
@kmcowan `ckpt-0` is the folder that houses the tensors when you download the model from the torrent. __Note__ it's 300GB.
[see scrot](https://private-user-images.githubusercontent.com/41754832/313725830-0c0f4ac0-bee4-40a0-a68c-4abe74369f4e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA3ODU2ODIsIm5iZiI6MTcxMDc4NTM4MiwicGF0aCI6Ii80MTc1NDgzMi8zMTM3MjU4MzAtMGMwZjRhYzAtYmVlNC00MGEwLWE2OGMtNGFiZTc0MzY5ZjRlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzE4VDE4MDk0MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFhYjM5ZWJjNDJiYjM1ZWU2Y2Q4MzJmODcyY2U4OGI5ZGQ3YWJkNDVmNzZhNjdhYTA0YTdkYTc2NDY0M2MzYWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Xuqv5v95__-jwKh2uKvrD96bq1vywJr-zZ99AnfpUXs)

**Reply by kmcowan** [reply]
Okay, that explains it. Thanks!
